app/views/documents/show/_contents.html.erb

```diff
@@ -1,7 +1,7 @@
 <% contents = @edition.document_type.contents.map { |field|
   {
     field: field.label,
-    value: render("documents/fields/#{field.type}", field: field, edition: @edition)
+    value: render("documents/fields/#{field.id}", field: field, edition: @edition)
   }
 } %>
```

Comment:
Rather than using a loop here I wonder if we could have the document_type return the hash, this would then allow multiple fields for a single thing (such as title and base_path as we recently discussed)